drivers/include/sdp3x.h

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2019 Dirk Ehmen
+ *               2020 Nishchay Agrawal
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_sdp3x SDP3x temperature and differential pressure sensor
+ * @ingroup     drivers_sensors
+ * @{
+ * @file
+ * @brief       Device driver interface for the SDP3x sensor.
+ *
+ * @author      Dirk Ehmen       <ehmen@ibr.cs.tu-bs.de>
+ * @author      Nishchay Agrawal <f2016088@pilani.bits-pilani.ac.in>
+ * @}
+ */
+
+#ifndef SDP3X_H
+#define SDP3X_H
+
+#include "saul.h"
+#include "xtimer.h"
+#include "mutex.h"
+#include "periph/i2c.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
+ *  Flags for sensor reading
+ */
+#define SDP3X_FLAG_MASS_FLOW  0x02      /* Use Mass Flow for Temperature Compensation */
+#define SDP3X_FLAG_DIFF_PRESS 0x00      /* Use Differential Pressure for Temperature Compensation */
+#define SDP3X_FLAG_AVERAGE    0x01      /* Average values */
+#define SDP3X_FLAG_NO_AVERAGE 0x00      /* Don't Average values */
```

Comment:
```suggestion
/**
 *  @brief   Flags for sensor reading
 * @{
 */
#define SDP3X_FLAG_MASS_FLOW  0x02      /**< Use Mass Flow for Temperature Compensation */
#define SDP3X_FLAG_DIFF_PRESS 0x00      /**< Use Differential Pressure for Temperature Compensation */
#define SDP3X_FLAG_AVERAGE    0x01      /**< Average values */
#define SDP3X_FLAG_NO_AVERAGE 0x00      /**< Don't Average values */
/** @} */
```

to make doxygen happy